cumulusci/core/tests/test_flowrunner.py

```diff
@@ -280,11 +278,6 @@ def test_init__nested_options(self):
         flow = FlowCoordinator(self.project_config, flow_config)
         self.assertEqual("bar", flow.steps[0].task_config["options"]["foo"])
 
-    def test_init_ambiguous_step(self):
-        flow_config = FlowConfig({"steps": {1: {"task": "None", "flow": "None"}}})
-        with self.assertRaises(FlowConfigError):
-            FlowCoordinator(self.project_config, flow_config, name="test")
-
```

Comment:
We'll want some version of this test here to verify the behavior is as expected.

Reply:
added test for ambiguous step to `test_utils_merge_config.py`